File: Calculator.cpp
#include<iostream>
using namespace std;
int main(){
    double num1, num2;
    char op;
    double result;

    cout << "Enter first number: ";
    cin >> num1;
    cout << "Enter second number: ";
    cin >> num2;

    cout <<"Choose an operation (+, -, *, /):";
    cin >> op;

    switch(op){
        case '+':
            result = num1 + num2 ;
            break;

        case '-':
            result = num1 - num2 ;
            break;

        case '*':
            result = num1 * num2 ;
            break;

        case '/':
            if(num2 != 0){
                result = num1 / num2 ;
            }else {
                cout << "Error: Division by zero is not allowed." << endl;
                return 1;
            }
            break;
        default:
            cout << "Error: Invalid operation." << endl;
            return 1;
    }
    cout << "The result is: " << result << endl;

    return 0;
}

File: Calculator.py
def main():
    
    num1 = int(input("Enter the first number : "))
    num2 = int(input("Enter the second number : "))
    
    print("Select operation:")
    print("1. Add")
    print("2. Subtract")
    print("3. Multiply")
    print("4. Divide")
    
    choice = input("Enter your choice (1/2/3/4) : ")
    
    if choice == "1":
        result = num1 + num2
        print(f"The result is : {result}")
    elif choice == "2":
        result = num1 - num2
        print(f"The result is : {result}")
    elif choice == "3":
        result = num1 * num2
        print(f"The result is : {result}")
    elif choice == "4":
        if num2 != 0:
            result = num1 / num2
            print(f"The result is : {result}")
        else:
            print("Error!!! Division by zero is not possible.")
    else:
        print("Invalid choice")

if __name__ == "__main__":
    main()
